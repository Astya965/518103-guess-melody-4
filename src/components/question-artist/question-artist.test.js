import React from "react";
import renderer from "react-test-renderer";

import QuestionArtist from "./question-artist.jsx";

const testQuestionArtist = {
  type: `artist`,
  song: {
    artist: `Aaron Dunn`,
    src: `https://upload.wikimedia.org/wikipedia/commons/f/fb/Aaron_Dunn_-_Sonata_No_1_-_Movement_1.ogg`,
  },
  answers: [{
    picture: `https://api.adorable.io/avatars/122`,
    artist: `Aaron Dunn`,
  }, {
    picture: `https://api.adorable.io/avatars/128`,
    artist: `Daniel Veesey`,
  }, {
    picture: `https://api.adorable.io/avatars/54`,
    artist: `Karine Gilanyan`,
  }],
};

it(`Render QuestionArtist`, () => {
  const tree = renderer.
    create(
        <QuestionArtist question={testQuestionArtist} />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
