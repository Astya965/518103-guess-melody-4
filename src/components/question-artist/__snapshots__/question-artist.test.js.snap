// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render QuestionArtist 1`] = `
<section
  className="game game--artist"
>
  <header
    className="game__header"
  >
    <a
      className="game__back"
      href="#"
    >
      <span
        className="visually-hidden"
      >
        Сыграть ещё раз
      </span>
      <img
        alt="Угадай мелодию"
        className="game__logo"
        src="img/melody-logo-ginger.png"
      />
    </a>
    <svg
      className="timer"
      viewBox="0 0 780 780"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        className="timer__line"
        cx="390"
        cy="390"
        r="370"
        style={
          Object {
            "filter": "url(#blur)",
            "transform": "rotate(-90deg) scaleY(-1)",
            "transformOrigin": "center",
          }
        }
      />
    </svg>
    <div
      className="game__mistakes"
    >
      <div
        className="game__mistakes"
      >
        <div
          className="wrong"
        />
        <div
          className="wrong"
        />
      </div>
    </div>
  </header>
  <section
    className="game__screen"
  >
    <h2
      className="game__title"
    >
      Кто исполняет эту песню?
    </h2>
    <div
      className="game__track"
    >
      <div
        className="track"
      >
        <button
          aria-label="Pause button"
          className="track__button track__button--pause"
          disabled={true}
          onClick={[Function]}
          type="button"
        />
        <div
          className="track__status"
        >
          <audio
            src="https://upload.wikimedia.org/wikipedia/commons/f/fb/Aaron_Dunn_-_Sonata_No_1_-_Movement_1.ogg"
          />
        </div>
      </div>
    </div>
    <form
      className="game__artist"
    >
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="Aaron Dunn"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="Aaron Dunn"
        />
        <label
          className="artist__name"
          htmlFor="Aaron Dunn"
        >
          <img
            alt="Aaron Dunn"
            className="artist__picture"
            src="https://api.adorable.io/avatars/122"
          />
          Aaron Dunn
        </label>
      </div>
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="Daniel Veesey"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="Daniel Veesey"
        />
        <label
          className="artist__name"
          htmlFor="Daniel Veesey"
        >
          <img
            alt="Daniel Veesey"
            className="artist__picture"
            src="https://api.adorable.io/avatars/128"
          />
          Daniel Veesey
        </label>
      </div>
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="Karine Gilanyan"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="Karine Gilanyan"
        />
        <label
          className="artist__name"
          htmlFor="Karine Gilanyan"
        >
          <img
            alt="Karine Gilanyan"
            className="artist__picture"
            src="https://api.adorable.io/avatars/54"
          />
          Karine Gilanyan
        </label>
      </div>
    </form>
  </section>
</section>
`;
